docs/set-up-your-environment.md

```diff
@@ -0,0 +1,104 @@
+---
+id: set-up-your-environment
+title: Set Up Your Environment
+hide_table_of_contents: true
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import constants from '@site/core/TabsConstants';
+
+import GuideLinuxAndroid from './\_getting-started-linux-android.md';
+import GuideMacOSAndroid from './\_getting-started-macos-android.md';
+import GuideWindowsAndroid from './\_getting-started-windows-android.md';
+import GuideMacOSIOS from './\_getting-started-macos-ios.md';
+
+In this guide, you'll learn how to set up your environment, so that you can run your project with Android Studio and Xcode. This will allow you to develop with Android emulators and iOS simulators, build your app locally, and more.
+
+:::note
+This guide requires Android Studio or Xcode. If you already have one of these programs installed, you should be able to get up and running within a few minutes. If they are not installed, you should expect to spend about an hour installing and configuring them.'
+
+<details>
+<summary>Is setting up my environment required?</summary>
+
+Setting up your environment is not required if you're using a Framework. With that, if you prefer to develop with Android Studio and/or Xcode, you can set up your local computer while still using a Framework.
```

Comment:
Link to the Getting Started page somewhere where we define what is a Framework?

Reply:
> With that, if you prefer to develop with Android Studio and/or Xcode, you can set up your local computer while still using a Framework.

@jonsamp the wording here is a bit confusing to me.

I would say something along: "With a React Native Framework, you don't need to setup Android Studio or XCode as a Framework will take care of building the native app for you. Therefore, you can use your web IDE of choice"

Reply:
I added an entry for "React Native Framework" here: architecture/glossary#react-native-framework

Let me know if that's the wrong place to add a term/definition like that.